packages/account/src/Styles/account.scss

```diff
@@ -1611,7 +1611,7 @@ $MIN_HEIGHT_FLOATING: calc(
             padding-left: 3%;
             list-style-type: disc;
 
-            @include mobile {
+            @include mobile-or-tablet-screen {
                 padding-left: 5%;
```

Comment:
@suisin-deriv @utkarsha-deriv shouldnt this be `padding-inline-start` ?

Reply:
Yes, let me update this.